charts/redis-ha/values.yaml

```diff
@@ -175,6 +176,8 @@ redis:
   #  limits:
   #    memory: 700Mi
 
+  lifecycle: {}
```

Comment:
Can you provide a link to people who potentially don't understand the lifecycle like: 

> Pod lifecycle command for cases of shutdown... etc etc https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/